Resources/ServerInfo/Guidebook/Engineering/Thermomachines.xml

```diff
@@ -0,0 +1,29 @@
+﻿<Document>
+  # Thermomachines
+  Thermomachines are devices that manipulate the temperature of gases within a pipe network or exposed atmosphere.
+  <Box>
+    <GuideEntityEmbed Entity="SpaceHeater" Caption=""/>
+    <GuideEntityEmbed Entity="GasThermoMachineHeater" Caption=""/>
+    <GuideEntityEmbed Entity="GasThermoMachineFreezer" Caption=""/>
+  </Box>
+  They are essential for maintaining the temperature of gasses for various applications.
+
+  All thermomachines work by using electrical power to preform work on the atmosphere to either heat or cool it. The amount of work they do is directly related to the amount of power they consume.
+
+  Thermomachines also have an efficiency coefficient, which determines how much work they can do per unit of power consumed.
```

Comment:
freezer thermomachines do not appear to be thermodynamically sound as per

            float dQ = thermoMachine.HeatCapacity * thermoMachine.Cp * args.dt;

with the "Cp" variable being the Coefficient of Performance, which is -3.9 lmfao, so I might just omit this, not sure how I feel about it.

I *do want* people to do nerd atmos math to find cool solutions to problems though...